_layouts/home.html

```diff
@@ -13,12 +13,99 @@ <h2 id="countdown">&nbsp;</h2>
+                              <!-- Slide Text Layer -->
+                              <div class="slide-text slide_style_left">
+                                <h1 data-animation="animated zoomInRight">Looking forward to meet you in our conference!!!</h1>
+                                <p data-animation="animated fadeInLeft">The best is yet to come!!!</p>
+                                <a href="http://bootstrapthemes.co/" target="_blank" class="btn btn-default" data-animation="animated fadeInUp">Become an host</a>
+                                <a href="http://bootstrapthemes.co/" target="_blank"  class="btn btn-primary" data-animation="animated fadeInDown">Attend an event</a>                              </div>
+                          </div>
+                      </div>
+                  </div>
+                  <!-- End of Slide -->
+
+                  <!-- Second Slide -->
+                  <div class="item">
+
+                      <!-- Slide Background -->
+                      <img src="../img/homepage_slide#2.jpg" alt="Bootstrap Touch Slider"  class="slide-image"/>
+                      <div class="bs-slider-overlay"></div>
+                      <!-- Slide Text Layer -->
+                      <div class="slide-text slide_style_center">
+                        <h1 data-animation="animated zoomInRight">Looking forward to meet you in our conference!!!</h1>
+                        <p data-animation="animated fadeInLeft">The best is yet to come!!!</p>
+                        <a href="http://bootstrapthemes.co/" target="_blank" class="btn btn-default" data-animation="animated fadeInUp">Become an host</a>
+                        <a href="http://bootstrapthemes.co/" target="_blank"  class="btn btn-primary" data-animation="animated fadeInDown">Attend an event</a>
+                      </div>
+                  </div>
+                  <!-- End of Slide -->
+
+                  <!-- Third Slide -->
+                  <div class="item">
+
+                      <!-- Slide Background -->
+                      <img src="../img/homepage_slide#3.jpg" alt="Bootstrap Touch Slider"  class="slide-image"/>
+                      <div class="bs-slider-overlay"></div>
+                      <!-- Slide Text Layer -->
+                      <div class="slide-text slide_style_right">
+                        <h1 data-animation="animated zoomInRight">Looking forward to meet you in our conference!!!</h1>
+                        <p data-animation="animated fadeInLeft">The best is yet to come!!!</p>
+                        <a href="http://bootstrapthemes.co/" target="_blank" class="btn btn-default" data-animation="animated fadeInUp">Become an host</a>
+                        <a href="http://bootstrapthemes.co/" target="_blank"  class="btn btn-primary" data-animation="animated fadeInDown">Attend an event</a>                      </div>
+                  </div>
+                  <!-- End of Slide -->
```

Comment:
currently the three slides have the same content, only the image would change. As already mentioned I'm not sure if we really need a slider. What do you think the content for each could be? I'm happy to have a slider, if it's actually making sense to display three different things.